src/pages/home/BuiltForSection/BuiltForSection.tsx

```diff
@@ -0,0 +1,68 @@
+import { t, Trans } from '@lingui/macro'
+import { Row } from 'antd'
+import { DEFAULT_HOMEPAGE_GUTTER } from '../Landing'
+import { SectionContainer } from '../SectionContainer'
+import { SectionHeading } from '../SectionHeading'
+import { BuiltForCard } from './BuiltForCard'
+
+export const BuiltForSection = () => {
+  return (
+    <SectionContainer>
+      <SectionHeading
+        heading={<Trans>Built for ideas like yours</Trans>}
+        subheading={
+          <Trans>
+            Juicebox is as versatile as you are, build anything from an NFT
+            project to a boutique crypto lawfirm and everything in between.
+          </Trans>
+        }
+      />
+      <Row gutter={DEFAULT_HOMEPAGE_GUTTER}>
```

Comment:
Is there a tailwind solution here?

Reply:
Are we trying to ditch `Row` and `Col` altogether?